assessments/guide/joint-review.md

```diff
@@ -43,31 +45,44 @@ from their repo into the self-assessment.
 
 ## Metadata
 
+### The review
+
+A table with specific information concerning the review of the project.
+
+|  |  |
+| -- | -- |
+| Project Security Lead | $GITHUB-HANDLE |
+| Security TAG Lead Reviewer | $GITHUB-HANDLE |
+| Additional Reviewers | $GITHUB-HANDLE, $GITHUB-HANDLE, etc. |
+| Date of Review Closure | $DATE review was submitted for PR |
+
+### The project
+
 A table at the top for quick reference information, later used for indexing.
 
 |   |  |
 | -- | -- |
 | Software | A link to the software’s repository.  |
 | Security Provider | Yes or No. Is the primary function of the project to support the
-security of an integrating system?  |
-| Languages | languages the project is written in |
-| SBOM | Software bill of materials.  Link to the libraries, packages, versions used by
- the project, may also included
-irect dependencies. |
+security of an integrating system?  | | Languages | languages the project is
```

Comment:
lol so actually the original file doesn't work, can't span tables over multiple lines, would need to be on the same line, the linter should be configured already to handle exceptions for this https://github.com/cncf/tag-security/blob/main/ci/lint-config.json#L5